### PR TITLE
Added a link on QPT landing page

### PR DESCRIPTION
**Pull request by viktym**
## Purpose of this pull request
 
Add a link on QPT landing page
 
## Affected Support KB pages
 

 
https://support.magento.com/hc/en-us/articles/360047139492-Magento-Quality-Patches-released-new-tool-to-self-serve-quality-patches

### CODE REVIEW

**Comment by almarchenko** [comment]
@Rinn22  please don't import yet

File: src/announcements/adobe-commerce-announcements/magento-quality-patches-released-new-tool-to-self-serve-quality-patches.md [hunk 1]
```diff
@@ -19,6 +19,6 @@ Having installed the tool, you can check for available patches, apply or revert
 
 Please refer to our developer documentation for details:
 
-* [QPT for Adobe Commerce on-premises](https://devdocs.magento.com/guides/v2.4/comp-mgr/patching/mqp.html)
-* [QPT for Adobe Commerce on cloud infrastructure](https://devdocs.magento.com/cloud/project/project-patch.html)
-* [QPT release notes](https://devdocs.magento.com/quality-patches/release-notes.html)
+* [Available patches in Quality Patches Tool](https://devdocs.magento.com/quality-patches/tool.html#patch-grid)
```
Comment (almarchenko) [inline src/announcements/adobe-commerce-announcements/magento-quality-patches-released-new-tool-to-self-serve-quality-patches.md:22]:
```suggestion
* [Patches available in Quality Patches Tool](https://devdocs.magento.com/quality-patches/tool.html#patch-grid)
```
```suggestion
* [Available patches in Quality Patches Tool](https://devdocs.magento.com/quality-patches/tool.html#patch-grid)
```

**Review by almarchenko — CHANGES_REQUESTED** [review]

File: src/announcements/adobe-commerce-announcements/magento-quality-patches-released-new-tool-to-self-serve-quality-patches.md (hunk 1 above)
Comment (almarchenko) [inline src/announcements/adobe-commerce-announcements/magento-quality-patches-released-new-tool-to-self-serve-quality-patches.md:22]:
```suggestion
* [Patches available in Quality Patches Tool](https://devdocs.magento.com/quality-patches/tool.html#patch-grid)
```

**Review by almarchenko — CHANGES_REQUESTED** [review]

**Review by almarchenko — APPROVED** [review]

**Comment by almarchenko** [comment]
@magento import code to magento-kb/knowledge-base

**Comment by m2-github-services** [comment]
@almarchenko the branch with the code is successfully imported into the `magento-kb/knowledge-base` repository. Branch name: `imported-viktym-knowledge-base-102`.

**Pull request merged**